system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -672,6 +670,13 @@ bool CLdapSecManager::authenticate(ISecUser* user)
         isUserCached = m_permissionsCache->lookup(*cachedUser);//populate cachedUser with cached values
     }
 
+    if (AS_AUTHENTICATED == user->getAuthenticateStatus())
+    {
+        if(isCaching && !isUserCached)
```

Comment:
Looks much better.

Is there any problem with not setting the digital signature in he cached entry?
If yes, what.
If no, why does the code further down (line 715) bother setting it?